src/screens/Services/Services.js

```diff
@@ -126,95 +125,116 @@ class ServicesScreen extends React.Component<Props> {
     if (isAltalixAvailable === null) loadAltalixInfo();
   }
 
-  getServices = (): Service[] => {
-    const {
-      navigation,
-      isActiveAccountSmartWallet,
-      isSmartWalletActivated,
-      accounts,
-    } = this.props;
+  navigateToArchanovaSupportedServiceScreen = (route: string) => {
```

Comment:
```suggestion
  navigateToRouteIfArchanovaWallet = (route: string) => {
```

Reply:
Changed to `navigateToRouteIfArchanovaWalletActivated` since it's actually a check if Archanova wallet is activated (deployed) and can use the service.